dask/dataframe/groupby.py

```diff
@@ -182,6 +182,10 @@ def _groupby_raise_unaligned(df, **kwargs):
         if isinstance(by, str):
             by = [by]
         kwargs.update(by=list(by))
+
+    if dropna is not None:
```

Comment:
Could you rather do this in the _nunique functions and keep kwargs here? That makes this a little less entangled

Reply:
Yeah sure, I will change it.